src/js/help.js

```diff
@@ -0,0 +1,111 @@
+        '<li>Full Word (Gurmukhi)' +
+        '<ul><li>You can type in the full word(s) in gurmukhi that you are searching for.</li></ul></li>' +
+        '<li>English Translation (Full Word)' +
+        '<ul><li>You can search for shabads via the english translations. For example, type in the word ' +
+        '"bird" and the results will show all shabads that include a translation for "bird".</li></ul>' +
+        '</li></ul>'],
+      [ 'I can\'t find my shabad.',
+        '<ul><li>' +
+        'STTM\'s database includes shabads from Sri Guru Granth Sahib Ji, Sri Dasam Granth Sahib, ' +
+        'Bhai Gurdas Jee, Bhai Gurdas Singh Jee, Bhai Nand Lal Jee, and various rehatnamas and ' +
+        'thankhanamas. If your shabad is not from one of those sources, it will not appear.</li>' +
+        '<li>If you verified your shabad should be in the database and it is still not coming up, ' +
+        'make sure you are typing in the correct letters for search. If you aren’t sure about a ' +
+        'letter, or are still having difficulties, try using one of the different search types to ' +
+        'locate the shabad.</li></ul>'],
+      [ 'How do I connect STTM to a projector?',
+        'Plug in your computer to the projector or TV. Once you do this, go to your computer\'s ' +
+        'display settings and change it to "Extended Desktop" (NOT mirroring). After that, launch ' +
+        'STTM and turn on "Presenter View" in the settings! ' +
+        '<img src="/assets/images/help/desktop-extend-pc.png" />' +
+        '<img src="/assets/images/help/desktop-extend-mac.png" />'],
+      [ 'How do I change the background color?',
+        'After launching STTM, click the icon for settings, and choose your theme.' +
+        '<img src="/assets/images/help/desktop-theme.png" />'],
+      [ 'Can you view the Gurbani in Larivaar?',
+        'Yes! After launching STTM, click the icon for settings, and choose the Larivaar option.' +
+        '<img src="/assets/images/help/desktop-larivaar.png" />'],
+      [ 'How can I make the fonts on the screen smaller?',
+        'After launching STTM, click the icon for settings, and scroll down for options to ' +
+        'adjust the font size. <img src="/assets/images/help/desktop-font-size.png" />'],
+      [ 'Where can I see my previous shabads?',
+        'When using STTM in “Presenter View”, your history will appear in the bottom right ' +
+        'quadrant. You can click any of the shabads in the list to bring them back up as your ' +
+        'primary one. <img src="/assets/images/help/desktop-history.png" />'],
+      [ 'How do I report a mistake?',
+        'Visit <a href="https://sikhitothemax.org">SikhiToTheMax.org</a> and click "Feedback" ' +
+        'at the bottom of the page.']
+    ]
+  };
+
+  const sidebar = (<div id="sidebar"></div>);
+  const main = (<main></main>);
+  const sidebarList = (<ul></ul>);
+  sidebar.appendChild(sidebarList);
```

Comment:
You can simply compose all this in JSX. I couldn't complete the entire thing, but this should help you understand how to do it.

```jsx
<div id='help'>
  <div id='sidebar'>
  </div>
  <main>
    {Object.keys(content).map(header => (
      <section id={header} key={header}>
        <h2>{header}</h2>
        <li>
          <a href={`#${header}`}>{header}</a>
          <ul>
            {
              content[header].map(([ firstItem ]) => {
                const title = firstItem.split(' ').join('-').toLowerCase().replace(/[^a-z.\-]/g, '').substr(0, 50);
                return (<li><a href={'#' + title}>{firstItem}</a></li>);
              })
            }
         </ul>
        
        {
           content[header].map(q => {
            const title = q[0].split(' ').join('-').toLowerCase().replace(/[^a-z.\-]/g, '').substr(0, 50);
            return (
              <div class='question' id={title}>
              </div>
            );
          })
        }
      </section>
    )}
  </main>
</div>
```